tests/countries/test_aland_Islands.py

```
@@ -0,0 +1,410 @@
+
+from unittest import TestCase
+
+from holidays.countries.aland_islands import AlandIslands, ALA, AX
+from tests.common import CommonCountryTests, SundayHolidays
+
+
+class TestAland(CommonCountryTests, SundayHolidays, TestCase):
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        
```

Ruff (0.8.2)

Blank line contains whitespace

Remove whitespace from blank line

(W293)